_data/componentlist.json

```diff
@@ -164,10 +164,10 @@
             "url": "/documentation/header",
             "text": "Header & Subnav",
             "type": "component",
-            "ux": 10,
-            "ui": 10,
+            "ux": 20,
+            "ui": 20,
```

Comment:
My thinking was that we have a functional header. With the change away from the workstream bugs, it looks weird to have all this code on the page and the HTML, CSS, and Angular logo greyed out. It's confusing.

So my thinking is that if we have HTML/CSS/NG code that people can use, those should be set to `20`. The HTML/CSS/NG logos don't really represent the workstreams anymore. They just show if we have code there, what type it is, and if we don't are we planning to add it.